ch2driver/pytpcc/drivers/nestcollectionsdriver.py

```diff
@@ -1270,12 +1537,25 @@ def runCH2Queries(self, duration, endBenchmarkTime, queryIterNum):
             # good randomness for different clients and different query iterations.
             #random.seed(self.client_id*9973 + queryIterNum*19997)
             #random.shuffle(ch2_queries_perm)
+            #logging.info(f"{self.schema=}")
+            #logging.info(f"{self.analyticalQueries=}")
+
             ch2_queries_perm = constants.CH2_QUERIES_PERM[self.client_id]
-            ch2_queries = (
-                constants.CH2_QUERIES_NON_OPTIMIZED
-                if bool(int(os.environ.get("UNOPTIMIZED_QUERIES", 0)))
-                else constants.CH2_QUERIES
-            )
+            if self.schema == constants.CH2_DRIVER_SCHEMA["CH2"]:
+                if self.analyticalQueries == constants.CH2_DRIVER_ANALYTICAL_QUERIES["HAND_OPTIMIZED_QUERIES"]:
+                    #logging.info("Using hand optimized CH2 queries")
+                    ch2_queries = constants.CH2_QUERIES
+                else:
+                    #logging.info("Using non-optimized CH2 queries")
+                    ch2_queries = constants.CH2_QUERIES_NON_OPTIMIZED
+            else:
+                if self.analyticalQueries == constants.CH2_DRIVER_ANALYTICAL_QUERIES["HAND_OPTIMIZED_QUERIES"]:
+                    #logging.info("Using hand optimized CH2++ queries")
+                    ch2_queries = constants.CH2PP_QUERIES
+                else:
+                    #logging.info("Using non-optimized CH2++ queries")
+                    ch2_queries = constants.CH2PP_QUERIES_NON_OPTIMIZED
```

Comment:
We can remove all the commented-out logging statements here before merging

Reply:
Will do this in a separate merge.